evm_v10/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "fil_actor_evm_v10"
+description = "Builtin EVM actor for Filecoin"
+version = "10.0.0"
+license = "MIT OR Apache-2.0"
+authors = ["Protocol Labs", "Filecoin Core Devs"]
+edition = "2021"
+repository = "https://github.com/filecoin-project/builtin-actors"
+keywords = ["filecoin", "web3", "wasm", "evm"]
+
+[lib]
+## lib is necessary for integration tests
+## cdylib is necessary for Wasm build
+crate-type = ["cdylib", "lib"]
+
+[dependencies]
+fil_actors_runtime_v10 = { workspace = true }
+fvm_shared = { workspace = true }
+serde = { workspace = true }
+serde_tuple = { workspace = true }
+num-traits = { workspace = true }
+num-derive = { workspace = true }
+cid = { workspace = true }
+anyhow = { workspace = true }
+log = { workspace = true }
+fvm_ipld_blockstore = { workspace = true }
+fvm_ipld_encoding = "0.3.3"
+multihash = { workspace = true }
+hex = { workspace = true }
+hex-literal = { workspace = true }
+frc42_dispatch = { workspace = true }
+fil_actors_evm_shared_v10 = { version = "10.0.0", path = "shared_v10" }
```

Comment:
This needs to be part of the workspace. (Or refactored out entirely.)

Reply:
moved